microsoft-edge/webview2/how-to/debug-visual-studio-code.md

```diff
@@ -153,6 +154,34 @@ If you're debugging Office Add-ins, open the add-in source code in a separate in
 ```
 
 
+<!-- ====================================================================== -->
+## Debug WebView2 UWP Apps
+
+1. Install a WebView2 Runtime version past `106.0.1370.34`.
+
+1. Set the registry key “HKEY_CURRENT_USER\Software\Policies\Microsoft\Edge\WebView2\AdditionalBrowserArguments” equal to “--remote-debugging-pipe”.  Verify the registry key is set in the editor and that it matches this:
+
+   ![Registry Key](./media/uwp-debugging-registry-key.png)
```

Comment:
It's good to add a screenshot of the tool here, but the bullet point above should also say what the tool name is and how to open it.